### DISCUSSION TITLE
Using funeralzone/valueobjects 0.5.0?

### DISCUSSION THREAD

**Discussion by mechadragon01** [osiset/laravel-shopify#666] [General]
Any chance on this package using funeralzone/valueobjects 0.5.0 instead of 0.4.5? 0.5.0 is compatible with PHP 8, but osiset/larvel-shopify requires 0.4.5 forcing me to use PHP 7.

**Comment by gnikyt** [comment]
PHP 8 support is in place since 3 days ago on v16
